### ISSUE TITLE
User is not able to navigate to CIS Scans page

### ISSUE THREAD

**Issue by sowmyav27** [rancher/rancher#24644]
**What kind of request is this (question/bug/enhancement/feature request):** bug

**Steps to reproduce (least amount of steps as possible):**
1. As an admin, deploy a cluster.
2. Add a standard user - User 1 as a project owner of one of the projects in the cluster say Default project.
3. Login as User 1.
4. Navigate to Cluster --> Tools --> CIS Scans page.

**Expected Result:**
Page loads without any error

**Actual Result:**
Error is seen as below

**Chrome:**
<img width="405" alt="Screen Shot 2019-12-16 at 9 35 57 AM" src="https://user-images.githubusercontent.com/26032343/71036451-96983580-20d2-11ea-940a-041776596f84.png">

**Firefox**
<img width="315" alt="Screen Shot 2019-12-16 at 9 35 51 AM" src="https://user-images.githubusercontent.com/26032343/71036470-a152ca80-20d2-11ea-8f6e-03d6b154befa.png">

**Note:**
- This does NOT happen always.
- same behavior is seen for a Project Owner, Cluster Member and a project member.

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): master-head
- Installation option (single install/HA): single

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): custom
- Kubernetes version (use `kubectl version`):

```
1.16
```

**Comment by sowmyav27** [comment]
Verified on latest master - commit : `` 
ui tag: latest2

Steps:
1. As an admin, deploy a cluster.
2. Add a standard user - User 1 as a project owner of one of the projects in the cluster say Default project.
3. Login as User 1.
4. Navigate to Cluster --> Tools --> CIS Scans page.
5. Run Scans is disabled for the user
<img width="1239" alt="Screen Shot 2019-12-23 at 11 12 01 AM" src="https://user-images.githubusercontent.com/26032343/71376122-570b9680-2575-11ea-990b-09f7066df0ed.png">

**Note:**
Verified for the following users:
Run Scans is disabled.
```
- Project owner
- CLuster member
- project member
- read only
```

Cluster owner is has access to run scans and list scans on the UI

**Comment by binarymist** [comment]
In Rancher v2.3.3 as admin there doesn't even appear to be the option Cluster --> Tools --> CIS Scans.
There is Cluster --> Tools, but no CIS Scans

Does this need to be enabled?

**Comment by leodotcloud** [comment]
@binarymist this would be available in 2.4 release.
